.github/workflows/scheduled_expo_build.yml

```
@@ -0,0 +1,24 @@
+name: Scheduled Expo Build
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+
+    steps:
+      - uses: actions/checkout@v3
```

What do you think of using the latest version for this step?

```suggestion
      - uses: actions/checkout@v4
```